lib/fluent/plugin/in_s3.rb

```diff
@@ -275,6 +279,9 @@ def create_sqs_client
       options[:region] = @s3_region if @s3_region
       options[:endpoint] = @sqs.endpoint if @sqs.endpoint
       options[:http_proxy] = @proxy_uri if @proxy_uri
+      if @sqs.aws_key_id && @sqs.aws_sec_key
+        options[:access_key_id] = @sqs.aws_key_id
+        options[:secret_access_key] = @sqs.aws_sec_key
```

Comment:
`end` is missing

Reply:
Sorry cleaned up to much, should work now